ci-operator/step-registry/stackrox/collector/update-support-packages/setup/stackrox-collector-update-support-packages-setup-commands.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+source .openshift-ci/jobs/update-support-packages/env.sh
+
+.openshift-ci/jobs/integration-tests/gcloud-init.sh
```

Comment:
This would conflict with the upcoming consolidation of CI logic #822, so probably makes sense either wait for it, or do a follow up PR right after.